README.md

```diff
@@ -6,7 +6,7 @@ Workflows used at [ZWEIDENKER](https://zweidenker.de) to build and deploy Flutte
 
 ```yaml
   quality:
-    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1
+    uses: zweidenker/flutter_workflows/.github/workflows/melos_quality_checks.yaml@v1.0.1
```

Comment:
How about we just say: Latest Version here and manually check? That way we don't need to adjust this for each release we do...

Reply:
'nWe could, but I feel like that would still lead to the same problem of copy'n'pasting the code, running it once and only then realizing, that you had to replace/update something in it.